sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -86,6 +86,18 @@ abstract class FileStreamSourceTest
     }
   }
 
+  case class AddTextFileDataWithSpaceInFileName(content: String, src: File, tmp: File)
```

Comment:
I would suggest that adding a new parameter to AddTextFileData rather than introducing a new class, such as
```  case class AddTextFileData(content: String, src: File, tmp: File, tempFilePrefix: String = "text")
```

Reply:
sure will update and add the test for file sink. Thanks for the review.